spring-boot-test-autoconfigure/src/main/resources/META-INF/spring.factories

```diff
@@ -45,6 +45,15 @@ org.springframework.boot.autoconfigure.jdbc.JdbcTemplateAutoConfiguration,\
 org.springframework.boot.autoconfigure.liquibase.LiquibaseAutoConfiguration,\
 org.springframework.boot.autoconfigure.transaction.TransactionAutoConfiguration
 
+# AutoConfigureJooqc auto-configuration imports
```

Comment:
It isn't a criticism on your submission, this is simply a general remark that given that we have to duplicate that _one more time_ perhaps it's time to introduce a dedicated concept for it. That would be part of this PR but I think it makes sense to have that discussion before merging the PR.